packages/shaderLab/src/parser/tokens/glslKeywords.ts

```diff
@@ -0,0 +1,31 @@
+import { createKeywordToken } from "./utils";
+
+// built-in variable
+export const GLPosition = createKeywordToken("gl_Position");
+export const GLFragColor = createKeywordToken("gl_FragColor");
+
```

Comment:
miss `gl_FragCoord, gl_FragDepth, gl_FragData` and so on,may be you can  refer to this document：https://registry.khronos.org/OpenGL/specs/es/3.0/GLSL_ES_Specification_3.00.pdf

Reply:
> miss `gl_FragCoord, gl_FragDepth, gl_FragData` and so on,may be you can refer to this document：https://registry.khronos.org/OpenGL/specs/es/3.0/GLSL_ES_Specification_3.00.pdf

Complementation of all glsl keywords is not the key aim of this pull request, plan to do it in next iteration.